src/CMakeLists.txt

```
@@ -69,7 +69,7 @@ if(ANDROID)
   set(FAST_TOKENIZER_FROM_SOURCES ON)
 else()
   set(FAST_TOKENIZERS_SUPPORTED ON)
-  if(USE_ABI0)
+  if(USE_ABI0 OR CMAKE_BUILD_TYPE MATCHES "Debug")
```

```suggestion
  if(USE_ABI0 OR WIN32 AND CMAKE_BUILD_TYPE MATCHES "Debug")
```
For non Windows platforms we can use release version of fast-tokenizers and debug OV tokenizers

---

Then we block a possibility to build 3rd parties on debug mode too.